config_panel.toml

```diff
@@ -10,8 +10,8 @@ services = ["__APP__-web", "__APP__-sidekiq", "__APP__-streaming"]
         [main.customization.max_toot_chars]
```

Comment:
I'm curious to know if you were able to successfully test this config panel

Theoretically one's supposed to define settings for each of these and use them in the template used during `ynh_add_config`, otherwise the changes made through the config panel are gonna be lost during each upgrade ... but then it's super boring as hell and madness to define tens of settings just for this ... Trying to confirm that this is a problem and finding a smart way to simplify all of this ...

Reply:
I didn't test this much and I assumed settings will be created once the config applied, but that doesn't solve the upgrade overwriting the config file
I was expecting the config panel to "re-apply" after an upgrade

Reply:
Yup unfortunately that's not the case :sweat_smile: 

Followup discussion : https://github.com/YunoHost/issues/issues/1973